assignments/closure.js

```diff
@@ -27,7 +43,22 @@ const counterMaker = () => {
 
 // ==== Challenge 4: Create a counter function with an object that can increment and decrement ====
 const counterFactory = () => {
+	counter = 0;
+	const counterObj = {
+		increment: function() {
+			return counter++;
```

Comment:
This logic doesn't quite work. You might want to put the double addition sign before the counter, same for the double subtraction sign